packages/terra-show-hide/src/ShowHideFocuser.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { forwardRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import { v4 as uuidv4 } from 'uuid';
+import styles from './ShowHide.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Text that precedes the focusable portion.
+   */
+  focusableText: PropTypes.string.isRequired,
+  /**
+   * The portion of the text that gets focus once the hidden portion is disclosed to be announced by assistive technologies
+   */
+  prefix: PropTypes.string,
+};
+
+/**
+ * @param str - string value
+ * @returns a trimmed string with trailing white space added
+ */
+const addTrailingWhiteSpace = (str) => `${str.trim()} `;
```

Comment:
We should have a better variable name than "str".

Reply:
Agreed, I'll rename it and update this comment with the commit # once it's done

Reply:
> We should have a better variable name than "str".

I renamed it to string as it is for basically any string value and I had hard time finding better option, but feel free to suggest a better name: https://github.com/cerner/terra-core/pull/3838/commits/c8f17c023738231996cc1c6baa69f7c03d4a046b, but feel free to suggest a better name.